config/locales/claims/en/layouts/application.yml

```diff
@@ -4,3 +4,21 @@ en:
       claims:
         header:
           service_name: Claim Funding for General Mentors
+  claims:
+    support:
+      schools:
+        new:
+          caption: Add organisation
+          cancel: Cancel
+          continue: Continue
+          title: Enter a school name, URN or postcode
+        check:
+          add_organisation: Add organisation
+          address: Address
+          cancel: Cancel
+          contact_details: Contact details
+          organisation_name: Organisation name
+          telephone: Telephone number
+          title: Check your answers
+          urn: Unique reference number (URN)
+          website: Website
```

Comment:
I think this can do into a separate locale file, `config/locales/claims/en.yml` at minimum, `config/locales/claims/en/support/schools.yml` if we want to be explicit.